integrationtesting/opendcs-tests/src/test/java/org/opendcs/dao/CwmsLocationLevelDAOTestIT.java

```diff
@@ -0,0 +1,286 @@
+package org.opendcs.dao;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.*;
+
+import java.util.Date;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opendcs.database.api.DataTransaction;
+import org.opendcs.database.api.OpenDcsDatabase;
+import org.opendcs.fixtures.AppTestBase;
+import org.opendcs.fixtures.annotations.ConfiguredField;
+import org.opendcs.fixtures.annotations.EnableIfDaoSupported;
+
+import decodes.cwms.CwmsLocationLevelDAO;
+import decodes.cwms.CwmsTimeSeriesDb;
+import decodes.tsdb.TimeSeriesDb;
+import opendcs.dai.LocationLevelDAI;
+import opendcs.dao.LocationLevelDAO;
+import opendcs.dao.LocationLevelDAO.LocationLevelValue;
+import opendcs.dao.LocationLevelDAO.LocationLevelSpec;
+
+/**
+ * Integration tests for CwmsLocationLevelDAO
+ * These tests require a real CWMS database connection
+ */
+public class CwmsLocationLevelDAOTestIT extends AppTestBase
+{
+    @ConfiguredField
+    private TimeSeriesDb tsDb;
+    
+    @ConfiguredField
+    private OpenDcsDatabase db;
+    
+    private CwmsLocationLevelDAO dao;
+    
+    @BeforeEach
+    public void setUp() throws Exception
+    {
+        if (tsDb instanceof CwmsTimeSeriesDb)
+        {
+            dao = new CwmsLocationLevelDAO((CwmsTimeSeriesDb) tsDb);
```

Comment:
This PR is to main, use the new style of stateless DAO and the `OpenDcsdatabase::getDao` method.